matchzoo/preprocessor/cdssm_preprocessor.py

```diff
@@ -93,9 +108,12 @@ def fit(self, inputs: typing.List[tuple]):
 
         # Store the fitted parameters in context.
         self.datapack.context['term_index'] = vocab_unit.state['term_index']
-        dim = len(vocab_unit.state['term_index']) + 1
-        self.datapack.context['input_shapes'] = [(None, dim * self._window),
-                                                 (None, dim * self._window)]
+        dim = (len(vocab_unit.state['term_index']) + 1)
+        self.datapack.context['dims'] = dim
```

Comment:
do not store useless variable into context, create a variable instead such as `self._dim_triletter`.